src/components/atoms/Modal.module.css

```diff
@@ -9,8 +9,9 @@
   left: 0;
   right: 0;
   bottom: 0;
-  background: var(--background-body-transparent);
-  backdrop-filter: blur(3px);
+  background: var(--brand-white);
+  box-shadow: 0 20px 25px -5px rgb(0 0 0 / 10%),
+    0 10px 10px -5px rgb(0 0 0 / 4%);
```

Comment:
question, nitpick(non-blocking): Are there any global variables that we could utilize here? These numbers look like "Pixel Perfect Positioning Numbers". It is great to follow a template but it is horrible for maintenance and consistency.